compiler/frontend/src/org/jetbrains/kotlin/diagnostics/PositioningStrategies.kt

```diff
@@ -940,6 +940,19 @@ object PositioningStrategies {
         }
     }
 
+    @OptIn(ExperimentalStdlibApi::class)
+    val COMMAS: PositioningStrategy<PsiElement> = object : PositioningStrategy<PsiElement>() {
+        override fun mark(element: PsiElement): List<TextRange> {
+            return buildList {
+                for (child in element.allChildren) {
+                    if (child.node.elementType == KtTokens.COMMA) {
+                        addAll(super.mark(child))
```

Comment:
It can be optimized since `mark` always returns only single element:

```suggestion
                        add(markSingleElement(child))
```